src/components/elements/MapPolygonPanel/ChecklistInformation.tsx

```diff
@@ -81,13 +83,26 @@ const ChecklistInformation = () => {
         <If condition={validationStatus}>
           <Then>
             {polygonValidationData.map((item, index) => (
-              <Text variant="text-14-light" key={index} className="flex items-center gap-2">
-                <Icon
-                  name={item.status ? IconNames.CHECK_PROGRESSBAR : IconNames.IC_ERROR_PANEL}
-                  className={`h-4 w-4 lg:h-5 lg:w-5 ${item.status ? "text-green-400" : ""}`}
-                />
-                {t(item.label)}
-              </Text>
+              <>
+                <Text variant="text-14-light" key={index} className="flex items-center gap-2">
+                  <Icon
+                    name={item.status ? IconNames.CHECK_PROGRESSBAR : IconNames.IC_ERROR_PANEL}
+                    className={`h-4 w-4 lg:h-5 lg:w-5 ${item.status ? "text-green-400" : ""}`}
+                  />
+                  {t(item.label)}
+                </Text>
+                {item.extra_info &&
+                  getFormatedExtraInfo(item.extra_info, item.id).map((info: any, index: number) => (
+                    <div className="flex items-start gap-[6px] pl-6" key={index}>
```

Comment:
typically is not a good practice to have the index as an id, can you use another id?